backend/src/polling_station/repository.rs

```diff
@@ -1,16 +1,12 @@
 use axum::extract::FromRef;
-use sqlx::{query, query_as, Sqlite, SqlitePool, Transaction};
+use sqlx::{query_as, SqlitePool};
 
+use crate::polling_station::structs::{PollingStation, PollingStationStatusEntry};
 use crate::AppState;
 
-use super::{
-    PollingStation, PollingStationResults, PollingStationResultsEntry, PollingStationStatusEntry,
-};
-
 pub struct PollingStations(SqlitePool);
 
 impl PollingStations {
-    #[cfg(test)]
```

Comment:
Needed to remove it to get this to work:
https://github.com/kiesraad/abacus/blob/62473a4e7add806a27ee03aea205007aa4005fe8/backend/src/polling_station/repository.rs#L235

I'm going to solve that in a better way by passing the `PollingStation` repository directly instead.

Reply:
Changed in 635e819432414e17ecc1138c9bd7f721fcbc8500.